L/libgsd/build_tarballs.jl

```diff
@@ -0,0 +1,51 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "libgsd"
+version = v"3.2.1"
+
+# Collection of sources required to complete build
+sources = [
+    ArchiveSource("https://github.com/glotzerlab/gsd/releases/download/v$(version)/gsd-$(version).tar.gz", "49a0e16276789b6953ece7e5a678bd22c128f8cc57cb967f4228ff6487770995")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+cd ${WORKSPACE}/srcdir/gsd-*
+mkdir -p "${libdir}"
+${CC} -std=c99 -fPIC gsd/gsd.c -shared -o "${libdir}/libgsd.${dlext}"
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = [
+    Platform("i686", "linux"; libc = "glibc"),
+    Platform("x86_64", "linux"; libc = "glibc"),
+    Platform("aarch64", "linux"; libc = "glibc"),
+    Platform("armv6l", "linux"; call_abi = "eabihf", libc = "glibc"),
+    Platform("armv7l", "linux"; call_abi = "eabihf", libc = "glibc"),
+    Platform("powerpc64le", "linux"; libc = "glibc"),
+    Platform("i686", "linux"; libc = "musl"),
+    Platform("x86_64", "linux"; libc = "musl"),
+    Platform("aarch64", "linux"; libc = "musl"),
+    Platform("armv6l", "linux"; call_abi = "eabihf", libc = "musl"),
+    Platform("armv7l", "linux"; call_abi = "eabihf", libc = "musl"),
+    Platform("x86_64", "macos"; ),
+    Platform("aarch64", "macos"; ),
+    Platform("x86_64", "freebsd"; )
+]
```

Comment:
```suggestion
platforms = supported_platforms()
platforms = filter!(!Sys.iswindows, platforms)
```
It is cleaner to filter out the platforms you don't want instead of listing everything when you have a lot of things.

(I think you just wanted to exclude Windows, right? It is also good to leave a comment explaining why it is excluded).

Reply:
Yes that is the idea. All of this has been created running the wizard and i have selected everything but windows. Therefore it might be worth updating this in the wizard as well. But thank you for the feedback.